systemd/manager.go

```diff
@@ -106,6 +106,14 @@ func (m *systemdUnitManager) Load(name string, u unit.UnitFile) error {
 	if err != nil {
 		return err
 	}
+	if len(u.Contents["Install"]) > 0 {
```

Comment:
I wonder, does that skip a present but empty `Install` section? I believe we get least surprise if it *doesn't* skip in this case. (Unless systemd has a problem with that?...)

Either way, it would be good to have a test for this case.